packages/builtin/array.pony

```diff
@@ -1,6 +1,139 @@
 class Array[A] is Seq[A]
   """
   Contiguous, resizable memory to store elements of type A.
+
+  # Usage
+  Creating an array of Strings, single line.
+  ```pony
+    let array: Array[String] = ["dog"; "cat"; "wombat"]
+  ```
+
+  Semicolon is an expression, so this is a multiline equivalent.
```

Comment:
the multiline equiv here can be removed. that's not about array, its about a language feature.